docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
   # Observation: the usage of this image of postgres is for testing purposes
  
   postgres: #We are using this standardized postgres image. No need to reinvent the wheel
-    image: debezium/postgres:13
+    image: debezium/postgres:16-alpine
```

Comment:
Just to let @VegSja and @febagni we changed the postgreSQL image to a newer one given some compatibility issues with Gio's Mac